Example/Standard Integration (Swift)/BrowseProductsViewController.swift

```diff
@@ -35,6 +35,7 @@ class BrowseProductsViewController: UITableViewController {
         super.viewDidLoad()
         self.navigationItem.title = "Emoji Apparel"
         self.navigationController?.navigationBar.isTranslucent = false
+        self.navigationController?.view.backgroundColor = .white
```

Comment:
This fixes the transition when `PaymentContext` pushes a small title view controller onto a large title nav controller (there's a black flash without this line).